server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,24 @@
+import express from "express";
+import { Review } from "../../../models/index.js";
+import cleanUserInput from "../../../../services/cleanUserInput.js";
+import { ValidationError } from "objection";
+
+const reviewsRouter = new express.Router({ mergeParams:true })
+
+reviewsRouter.post("/", async (req,res) =>{
```

Comment:
This isn't just a Reviews router, but instead a _nested_ router for `serviceReviews`